src/frontend/js/modules/header/profile-menu/profile-menu.controller.js

```diff
@@ -25,5 +26,18 @@
         self.avatarURL = esnAvatarUrlService.generateUrlByUserId(user._id, true);
       }
     }
+
+    function _accountUrl() {
+      const appGridItems = window.openpaas && window.openpaas.APP_GRID_ITEMS || process.env.APP_GRID_ITEMS;
+
+      if (appGridItems) {
+        const applications = JSON.parse(appGridItems);
+        const accountApp = applications.find(({ name }) => name === ACCOUNT_APPLICATION_NAME);
+
+        if (accountApp) return accountApp.url;
+      }
+
+      return window.openpaas && window.openpaas.ACCOUNT_SPA_URL || '/account/';
+    }
```

Comment:
I think the behavior should be to use `ACCOUNT_SPA_URL` prior to the account url found in `APP_GRID_ITEMS`.
If both are found the `ACCOUNT_SPA_URL` should be used.